Add experimental Scrimba skill test embeds to learn area (#39917)
mdn/content: files/en-us/learn_web_development/core/css_layout/test_your_skills/flexbox/index.md

sidebar: learnsidebar
---

> The aim of this skill test is to assess whether you understand how [flexbox and flex items](/en-US/docs/Learn_web_development/Core/CSS_layout/Flexbox) behave. Below are four sets of design problems that you might use flexbox to solve. Your task is to fix the problems.

Reviewer: nit to smooth it out slightly
```suggestion
The aim of this skill test is to assess whether you understand how [flexbox and flex items](/en-US/docs/Learn_web_development/Core/CSS_layout/Flexbox) behave. Below are four sets of design problems you can solve using flexbox. Your task is to fix the problems.
```
or "problems for which you can use flexbox"
or "problems that you can fix using flexbox" (to keep "that" usage")
